src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE = 40;
+  const sum = days * PRICE;
+
+  if (days >= 7) {
+    return sum - 50;
```

Comment:
The number '7' and the discount value '50' are hardcoded. Use constants with descriptive names to represent these values, enhancing the readability and maintainability of the code.